server.py

```diff
@@ -59,6 +58,12 @@ def __iter__(self):
 async def root():
     return root_template.render(all_requests=reversed(list(persistence)))
 
+def format_body(body):
+    gen = hexdump.hexdump(io.BytesIO(body), result='generator')
+    visual = list(gen)[:BODY_MAX_LINES_SHOWED]
```

Comment:
It's weird that the results are "generated" only to create a list from them. I understand why: to have lines! But a comment should be added to help others.